apps/web/styles/prosemirror.css

```diff
@@ -200,4 +200,12 @@ div[data-youtube-video] > iframe {
   div[data-youtube-video] > iframe {
     max-height: 100px;
   }
+}
+
+span[style] > strong {
+  color: inherit;
+}
+
+mark[style] > strong {
+  color: inherit;
```

Comment:
hey thanks for the change, it would be great if you can also add the changes to the prosemirror docs file

Reply:
> hey thanks for the change, it would be great if you can also add the changes to the prosemirror docs file

Can you link the doc file for prosemirror. Can't find anything related to prosemirror at https://github.com/steven-tey/novel/tree/main/apps/docs

Reply:
hey should be here
https://novel.sh/docs/guides/tailwind/setup#prosemirror-styles
https://github.com/steven-tey/novel/blob/main/apps/docs/guides/tailwind/setup.mdx#add-styles

Reply:
> hey should be here https://novel.sh/docs/guides/tailwind/setup#prosemirror-styles https://github.com/steven-tey/novel/blob/main/apps/docs/guides/tailwind/setup.mdx#add-styles

Updated the docs